src/squlearn/util/decompose_to_std.py

```diff
@@ -0,0 +1,34 @@
+from qiskit.circuit.library import standard_gates
+from qiskit.circuit import QuantumCircuit
+
+
+def decompose_to_std(circuit: QuantumCircuit, gate_list: list = []) -> QuantumCircuit:
+    """
+    Function to decompose the circuit to standard gates.
+
+    Args:
+        circuit (QuantumCiruit): The Circuit, which is supposed to be decomposed.
+        gate_list (list): List of gates, which are considered as standard gates. If the gate is not in the list, it will be decomposed. If the list is empty, all gates from qiskit.circuit.library.standard_gates and ["cx","cy","cz","measure"] are considered as standard gates.
+
+    Returns:
+        QuantumCiruit: The decomposed circuit.
```

Comment:
```suggestion
        circuit (QuantumCircuit): The Circuit, which is supposed to be decomposed.
        gate_list (list): List of gates, which are considered as standard gates. If the gate is
            not in the list, it will be decomposed. If the list is empty, all gates from
            `qiskit.circuit.library.standard_gates` and `["cx","cy","cz","measure"]` are considered
            as standard gates.

    Returns:
        QuantumCircuit: The decomposed circuit.
```